ci-operator/step-registry/ipi/metrics/ipi-metrics-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Updating Cluster Infra ConfigMap with the start and end times."
+START_TIME=$(cat "$SHARED_DIR/CLUSTER_INSTALL_START_TIME")
+END_TIME=$(cat "$SHARED_DIR/TEST_TIME_INSTALL_END")
+CLUSTER_INSTALL_PATCH=$(echo '{"data":{"startTime":"'$START_TIME'","endTime":"'$END_TIME'"}}')
```

Comment:
The `echo` is redundant.
```suggestion
CLUSTER_INSTALL_PATCH='{"data":{"startTime":"'$START_TIME'","endTime":"'$END_TIME'"}}'
```

Reply:
Turns out there is a shellcheck rule violation without the echo. "Quotes\backslashes will be treated as such"